akka-http-tests/src/test/scala/akka/http/scaladsl/marshalling/MarshallingSpec.scala

```diff
@@ -202,7 +202,10 @@ class MarshallingSpec extends FreeSpec with Matchers with BeforeAndAfterAll with
   override def afterAll() = TestKit.shutdownActorSystem(system)
 
   protected class FixedRandom extends java.util.Random {
-    override def nextBytes(array: Array[Byte]): Unit = "my-stable-boundary".getBytes("UTF-8").copyToArray(array)
+    override def nextBytes(array: Array[Byte]): Unit = {
```

Comment:
It seems for some reason the implicit conversion `copyArrayToImmutableIndexedSeq` to `IndexesSeq[Byte]` that introduces the `IterableOnceOps` that contain `copyToArray(array)` does not trigger automatically for some reason - perhaps due to ambiguous implicits somewhere?

`copyToArray(array, start, length)` doesn't have that problem since it's defined on `ArrayOps`, which has the additional benefit that it doesn't copy the array twice (`copyArrayToImmutableIndexedSeq` has been deprecated because it copies the array, but that might not always be necessary, like here).

Reply:
Thanks for the explanation.